src/coprocessor/statistics/cmsketch.rs

```diff
@@ -6,6 +6,7 @@ use murmur3::murmur3_x64_128;
 /// Refer:[Count-Min Sketch](https://en.wikipedia.org/wiki/Count-min_sketch)
 #[derive(Clone)]
 pub struct CmSketch {
+    #[allow(dead_code)]
```

Comment:
Can these be deleted?

Reply:
In this case, the `depth` is used in tests, so I'll keep it. Will clean up other cases.